### ISSUE TITLE
When I reboot the phone, I need to jailbreak again, otherwise I won't be able to open Untether.

### ISSUE THREAD

**Issue by Vn6958** [LinusHenze/Fugu14#135]
When I reboot the phone, I need to jailbreak again, otherwise I won't be able to open Untether.
Me iphone11, iOS version 14.5.1
Can we solve this problem?

**Comment by chillhahn** [comment]
Same issue for me iphone 11pro and ios 14.4

**Comment by Pagaman1** [comment]
iphone 11 pro on ios 14.4
After reboot, the phone is not Jailbroken.
I try it for 10 times with Unc0ver app and also with Untether app, I get the same result, the iphone reboot not Jailbroken.

**Comment by Einstein2150** [comment]
This is why Fugu14 is called in Germany a Mogelpackung. Fugu14 is not untethered - it is semi-untethered 🤫

**Comment by c0repwn3r** [comment]
> This is why Fugu14 is called in Germany a Mogelpackung. Fugu14 is not untethered - it is semi-untethered 🤫

Incorrect, Fugu14 is fully untethered. u0 is not. u0 requires two reboots to detect the untether sometimes, the 7.0.x releases were poorly made

**Comment by chillhahn** [comment]
i jailbreak it 4 times and made every time a reboot but still not jailbroken after reboot.🤷‍♂️

**Comment by Louiswu2011** [comment]
Same issue here iPhone 11 Pro on iOS 14.4

**Comment by coredoesdev** [comment]
> > Incorrect, Fugu14 is fully untethered. u0 is not. u0 requires two reboots to detect the untether sometimes, the 7.0.x releases were poorly made
> 
> i jailbreak it 4 times and made every time a reboot but still not jailbroken after reboot.🤷‍♂️

Because unc0ver is not untethered.
